python/taichi/ui/scene.py

```diff
@@ -142,7 +142,10 @@ def mesh(self,
         if vertex_count is None:
             vertex_count = vertices.shape[0]
         if index_count is None:
-            index_count = indices.shape[0]
+            if indices is None:
+                index_count = vertex_count  # FIXME : Need to confirm
+            else:
+                index_count = indices.shape[0]
```

Comment:
probably the following?
```python
if indices is not None:
    if index_count is None:
        index_count = indices.shape[0]
```

Reply:
I think we must pass a value to index_count.

Reply:
I tried to pass this value using None, but failed to get an error information. It's the reason that pybind donnot support to passing None value to int object.